app/views/users/show.html.erb

```diff
@@ -41,7 +41,14 @@
           Gi / No-gi
         <% end %>
       </p>
-      <%# TODO: Add distance %>
+      <% unless @user.id == current_user.id %>
+        <% if !current_user.latitude.nil? && !@user.latitude.nil? %>
+          <p>
+            <strong>Distance:</strong>
+            <%="#{@user.distance_to([current_user.latitude, current_user.longitude]).round(1)}"%> Km
+          </p>
+        <% end %>
+      <% end %>
```

Comment:
😨 Sorry, I had this in another branch but got merged into this one somehow. This is just to show the distance from the figma file:

<img width="618" alt="Screen Shot 2022-12-10 at 10 33 14 AM" src="https://user-images.githubusercontent.com/108884517/206862828-ae1a492d-0fb1-4a3f-9e6e-5f8be3f2d540.png">